### PR TITLE
CEDS-2659 Upgrade SBT to 1.3.6

### PR DESCRIPTION
**Pull request by MaciejRewera**
This version seems to be the last reliable version.
Higher versions contain some issues with importing a project. That would
need additional steps and workarounds to make it work.

### CODE REVIEW

**Comment by MaciejRewera** [comment]
Test this please.

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-resolvers += "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
+resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
 
 resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.9.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.10.0")
 
-addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.2")
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
```
Comment (PatrykRudnicki) [inline project/plugins.sbt:9]: Not sure if this should be a dependency inside the project. Didn't know that we had this in the project.

This is a kind of dependency that should probably leave in the global configuration for everyone, not inside the project

**Review by PatrykRudnicki — DISMISSED** [review]

File: project/plugins.sbt (hunk 1 above)
Comment (MaciejRewera) [reply project/plugins.sbt:9]: Did not think about it, just updated all plugins to the newest version. I'll have a look if it can be removed.

Comment (PatrykRudnicki) [reply project/plugins.sbt:9]: Yeah, I know, that's the reason I approved this :)

This is just to generate the dependency graphs. Codebase is not using it on 100%

**Review by PatrykRudnicki — APPROVED** [review]

**Pull request merged**